src/localization/strings/en.js

```diff
@@ -55,34 +73,25 @@ export const en = {
   "This is a CORRECT skype": "This is a CORRECT skype",
   "This is a CORRECT time": "This is a CORRECT time",
   "This is a CORRECT title": "This is a CORRECT title",
+  "Title is a required field": "Title is a required field",
+  "Title": "Title",
+  "To do": "To do",
   "User Name": "User Name",
-  "User Pane": "User Pane",
+  "User Panel": "User Pane",
   "User Profile": "User Profile",
   "User was updated successfully": "User was updated successfully",
   "User": "User",
   "View": "View",
   "We send a verify letter to your email!": "We send a verify letter to your email!",
   "Well done": "Well done",
-  "comment is a required field": "email is a required field",
-  "comment must be at least 3 characters": "comment must be at least 3 characters",
-  "email is a required field": "email is a required field",
-  "email must be a valid email": "email must be a valid email",
-  "name can have maximum 40 characters": "name can have maximum 40 characters",
-  "name is a required field": "name is a required field",
   "of the past and focus on the present!": "of the past and focus on the present!",
-  "password is a required field": "password is a required field",
-  "password must be at least 6 characters": "password must be at least 6 characters",
-  "phone can have maximum 20 characters": "phone can have maximum 20 characters",
-  "priority is a required field": "priority is a required field",
-  "rating is a required field": "rating is a required field",
-  "title is a required field": "title is a required field",
 
   // Week days
-  "Monday": "Monday",
-  "Tuesday": "Tuesday",
-  "Wednesday": "Wednesday",
-  "Thursday": "Thursday",
-  "Friday": "Friday",
-  "Saturday": "Saturday",
-  "Sunday": "Sunday",
+  "Monday": "Mon",
```

Comment:
The main rule

Your keys should equal english translations

"Monday" !== "Mon"

Reply:
And you've added it, but on Calendar we still using slice(0, 3), please fix)